### DISCUSSION TITLE
Code snippet ugly after migration to storybook 7

### DISCUSSION THREAD

**Discussion by Azcanba** [storybookjs/storybook#21937] [General]
Hello,

I migrated storybook from v6.5 to v7. and my code snippets are quite ugly :)

I am exporting a multiple button variants story in button.stories.tsx
![image](https://user-images.githubusercontent.com/128691736/230018152-2305f899-4bbc-4a50-8a85-47c90e32ff4d.png)

and when I open a code snippet for the code above in storybook. It looks kinda ugly 
![image](https://user-images.githubusercontent.com/128691736/230018469-b683a192-d69d-41a9-a4e1-54b1851ad0a7.png)

I would expect it to work same as before and see the exact code like on first picture also in code snippet.
Does anybody else have the similar issue ? There might be any parameters.docs.source... parameter which I dont know about ?

Thanks for answers

**Comment by grzegorznyga** [comment]
@Azcanba I have the same issue, did you find a solution?

**Comment by JReinhold** [comment]
This sounds like the bug reported here: https://github.com/storybookjs/storybook/issues/21747

**Reply by bratanon** [reply]
Still the same issue with 7.0.7 which includes that fix.
